src/app/ubs/ubs-admin/components/ubs-admin-sidebar/ubs-admin-sidebar.component.ts

```diff
@@ -1,19 +1,147 @@
-import { AfterViewInit, Component } from '@angular/core';
+import { AfterViewInit, Component, OnInit } from '@angular/core';
 import { BreakpointObserver } from '@angular/cdk/layout';
 import { UserMessagesService } from '../../../../ubs/ubs-user/services/user-messages.service';
 import { UbsBaseSidebarComponent } from 'src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component';
 import { JwtService } from '@global-service/jwt/jwt.service';
 import { listElementsAdmin } from '../../../ubs/models/ubs-sidebar-links';
+import { UbsAdminEmployeeService } from 'src/app/ubs/ubs-admin/services/ubs-admin-employee.service';
+import {
+  employeePositionsName,
+  AdminSideBarMenu,
+  EnablingAuthorities,
+  SideMenuElementsNames
+} from 'src/app/ubs/ubs-admin/models/ubs-admin.interface';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
 
 // @ts-ignore
 @Component({
   selector: 'app-ubs-admin-sidebar',
   templateUrl: './ubs-admin-sidebar.component.html'
 })
-export class UbsAdminSidebarComponent extends UbsBaseSidebarComponent implements AfterViewInit {
+export class UbsAdminSidebarComponent extends UbsBaseSidebarComponent implements AfterViewInit, OnInit {
   public listElementsAdmin = listElementsAdmin;
+  public employeeAuthorities: string[];
+  public employeePositions: string[];
+  public positionName: Array<string>;
+  public destroySub: Subject<boolean> = new Subject<boolean>();
 
-  constructor(public service: UserMessagesService, public breakpointObserver: BreakpointObserver, public jwtService: JwtService) {
+  private viewersArr = [
+    employeePositionsName.SuperAdmin,
+    employeePositionsName.Admin,
+    employeePositionsName.CallManager,
+    employeePositionsName.ServiceManager,
+    employeePositionsName.Logistician,
+    employeePositionsName.Navigator
+  ];
+
+  private employeesCertViewerArr = [
+    employeePositionsName.SuperAdmin,
+    employeePositionsName.Admin,
+    employeePositionsName.CallManager,
+    employeePositionsName.ServiceManager,
+    employeePositionsName.Logistician
+  ];
+
```

Comment:
viewersArr = [...employeesCertViewerArr,   employeePositionsName.Navigator]

Reply:
Thank you. Fixed it.